packages/atomic/src/components/search/facets/atomic-automatic-facet-generator/atomic-automatic-facet-generator.tsx

```diff
@@ -33,12 +34,15 @@ export class AtomicAutomaticFacetGenerator implements InitializableComponent {
   @InitializeBindings() public bindings!: Bindings;
   @State() public error!: Error;
 
+  public automaticFacetGenerator!: AutomaticFacetGenerator;
   @BindStateToController('automaticFacetGenerator')
   @State()
   private automaticFacetGeneratorState!: AutomaticFacetGeneratorState;
 
-  public automaticFacetGenerator!: AutomaticFacetGenerator;
   public searchStatus!: SearchStatus;
+  @BindStateToController('searchStatus')
+  @State()
+  public searchStatusState!: SearchStatusState;
```

Comment:
I added a test called 'should display nothing when response is empty'. 

IMO, I think it is clear that it is needed since, the searchStatusState is used inside of the render function and affect what is displayed. I guess, for most components, every state that is used in a condition in the render function needs to be listened to.